source/fuzz/fuzzer.cpp

```diff
@@ -217,12 +219,18 @@ Fuzzer::FuzzerResult Fuzzer::Run() {
                                                       initial_facts_);
 
   RepeatedPassInstances pass_instances{};
+
+  // The following passes are likely to be very useful: many other passes
+  // introduce synonyms, irrelevant ids and constants that these passes can work
+  // with.  We thus enable them with high probability.
+  MaybeAddRepeatedPass<FuzzerPassObfuscateConstants>(90, &pass_instances);
+  MaybeAddRepeatedPass<FuzzerPassApplyIdSynonyms>(90, &pass_instances);
+  MaybeAddRepeatedPass<FuzzerPassReplaceIrrelevantIds>(90, &pass_instances);
```

Comment:
I wonder if it's better to put these into the loop below. The motivation is that some of the transformations applied there might introduce instructions that we can later obfuscate, create synonyms to and replace with irrelevant ids.

In particular, I believe we don't have any irrelevant ids initially.

Reply:
MaybeAddRepeatedPass just makes the pass exist in pass_instances.

There is then, later, a loop that selects random passes from the sequence of pass indices, so it's not the case that these passes will be run first (necessarily).

See what I mean?

Reply:
Yes, you are right indeed. Sorry for the confusion.